project/GithubEnv.scala

```diff
@@ -0,0 +1,30 @@
+import scala.util.Properties
+import sbt.url
+import java.net.URL
+
+
+// https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/variables#default-environment-variables
+object GithubEnv {
+  lazy val repositoryVar: Option[(String, String)] = envVar("GITHUB_REPOSITORY")
+  lazy val runIdVar: Option[(String, String)] = envVar("GITHUB_RUN_ID")
+  lazy val shaVar: Option[(String, String)] = envVar("GITHUB_SHA")
+  lazy val workflowVar: Option[(String, String)] = envVar("GITHUB_WORKFLOW")
+
+  lazy val runUrl: Option[(String, URL)] =
+    for {
+      (_, repository) <- repositoryVar
+      (_, runId) <- runIdVar
+    } yield "GITHUB_RUN" -> url(s"https://github.com/$repository/actions/runs/$runId")
```

Comment:
Just a detail: I used natural language labels for links on the Scala 2 version

<img width="454" alt="image" src="https://github.com/user-attachments/assets/0682be50-4d0d-48e6-89cc-fa079ba5b09e">